### PR TITLE
barcode scanning looks up attendee info w/Android app

### PR DESCRIPTION
**Pull request by binary1230**
I even made a 1 minute video explaining this one: https://www.youtube.com/watch?v=-sh_wFqj_Sk

This is Really Cool.  I didn't think this was possible but apparently it's easy.  Direct barcode scanning right from ubersystem.

You have to have an app called XZing barcode scanner on android.

Once installed it can answer URI's on pages that start with ```xzing://```. When you click on that link/button, it'll open the barcode scanner on your app, scan the barcode, and you get the barcode data back in javascript.  From that point you can do whatever you want.

This is a proof of concept that shows how to get at this, it scans a printed barcode and pulls up a link to the attendee information via an AJAX request.

Since we know Chrome on Android can do this, it's reasonable to assume we could write our own URI handler for a locally running barcodescanner+webcam to make this run on reg laptops, or wherever is needed.

Short version is: this hardware interface stuff was what I thought would be the hard/annoying part in doing barcode stuff, however, it might actually be pretty trivial in the end, which is great.

I'm going to merge this since it affects nothing and I want to use it to test the servers to make sure the barcode key/seret config is solid.  Please review though, just know it's a little sloppy because it's a proof of concept.

### CODE REVIEW

File: barcode_utils.py [hunk 1]
```diff
@@ -190,7 +190,11 @@ def _barcode_raw_decrypt(value, key):
     # skip32 generates 4 bytes output from 4 bytes input
     _encrypt = False
     decrytped = bytearray(decoded)
-    skip32.skip32(key, decrytped, _encrypt)
+
+    try:
+        skip32.skip32(key, decrytped, _encrypt)
+    except Exception as e:
+        raise ValueError("failed to decrypt barcode. check secret_key, event_id, and whether this barcode is from this event") from e
```
Comment (binary1230) [inline barcode_utils.py]: this error usually means that your key is wrong, as I discovered by accident that my production key and dev key were different : )

**Comment by EliAndrewC** [comment]
+1

**Comment by binary1230** [comment]
as actually pretty cool as this was, I did find an even better solution for it now:
http://serratus.github.io/quaggaJS/examples/

same deal except it works with any system camera / webcam / phone, they have some demos which were working great with our encodings.

still, this PR was useful in getting the backend working.
